packages/chart-advisor/src/chart-advisor/index.ts

```diff
@@ -21,7 +23,7 @@ export class ChartAdvisor {
    */
   advise(params: AdviseParams) {
     const { data, dataProps, fields, options } = params;
-    const advices = this.advisor.advise({ data, dataProps, fields, options } as ChartAdviseParams);
+    const advices = this.advisor.advise({ data, dataProps, fields, options } as AdviseParams);
```

Comment:
could change to `this.advisor.advise(params)`

Reply:
done